python/src/spark_rapids_ml/core.py

```diff
@@ -438,7 +444,11 @@ def _train_udf(pdf_iter: Iterator[pd.DataFrame]) -> pd.DataFrame:
                     if multi_col_names:
                         features = pdf[multi_col_names]
                     else:
-                        features = np.array(list(pdf[alias.data]), order=array_order)
+                        # it is faster to convert to numpy array and then to cupy array then directly
+                        # invoking cupy array on the list
```

Comment:
Is this from observation or from some reference?  If a reference, might be nice to add link (so we can see if this ever changes in the future).

Also, a minor nitpick: "then to cupy array than..."

Reply:
Yeah. Also curious about the observation. More information/explanation would be helpful.

Reply:
It is from observation.  Will clarify.